doc/api/test.md

```diff
@@ -2814,6 +2814,11 @@ are defined, while others are emitted in the order that the tests execute.
         numbers and the number of times they were covered.
         * `line` {number} The line number.
         * `count` {number} The number of times the line was covered.
+    * `thresholds` {Object} An object containing whether or not the coverage for
+      each coverage type.
+      * `function` {boolean} Whether or not the coverage surpassed the `function` threshold.
+      * `branch` {boolean} Whether or not the coverage surpassed the `branch` threshold.
+      * `line` {boolean} Whether or not the coverage surpassed the `line` threshold.
```

Comment:
It might be useful to include the actual threshold values so that reporters have easy access to them.

Reply:
Do you mean like: `{ threshold, actual }` or just `threshold`? (`actual` can be gotten via `.totals.coveredXYZPercent`?

Reply:
I would provide the threshold values here. As you said, we already have the actual values. Whether or not we include these boolean values doesn't matter much to me.